maestro-cli/src/main/java/maestro/cli/runner/TestRunner.kt

```diff
@@ -0,0 +1,77 @@
+package maestro.cli.runner
+
+import maestro.Maestro
+import maestro.orchestra.NoInputException
+import maestro.orchestra.SyntaxError
+import maestro.orchestra.yaml.YamlCommandReader
+import org.fusesource.jansi.Ansi
+import org.fusesource.jansi.AnsiConsole
+import java.io.File
+import kotlin.concurrent.thread
+
+object TestRunner {
+
+    fun run(
+        maestro: Maestro,
+        flowFile: File,
+        continuous: Boolean,
+    ): Int {
+        AnsiConsole.systemInstall()
```

Comment:
Thinking about it, those 2 lines should probably be a part of the `ResultView`

Reply:
Updated